### ISSUE TITLE
TPC-H schema contains many extra indexes 

### ISSUE THREAD

**Issue by mariadb-DmitryVolkov** [cmu-db/benchbase#233]
I've noticed that TPC-H  schema contains many extra indexes (for quite some tables). Here is an example:
line item table has at least 6 extra indexes:
 
https://github.com/cmu-db/benchbase/blob/main/src/main/resources/benchmarks/tpch/ddl-postgres.sql#L125-L130

Where are these indexes came from? I've tried to read the tpc-h description but couldn't find one. I must be blind today. 

Looking at HammerDB, for example (this is just another popular tool, not a gold standard, of course) 
https://github.com/TPC-Council/HammerDB/blob/master/src/mariadb/mariaolap.tcl#L194-L215

it has some indexes, but not 6.

**Comment by apavlo** [comment]
@mariadb-DmitryVolkov Good point. We should look this over. All the different DBMS-specific DDL files have different indexes.
